database/20200506_create-schema.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE users (
+    id SERIAL NOT NULL PRIMARY KEY,
+    password TEXT NOT NULL,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    twitter VARCHAR,
+    phone VARCHAR,
+    email VARCHAR
+);
+
+CREATE TABLE github_info (
+    id SERIAL NOT NULL PRIMARY KEY,
+    user_id SERIAL NOT NULL UNIQUE REFERENCES users(id),
```

Comment:
I'm not sure if this will make a difference, the create statement i used for user_id was `user_id INTEGER UNIQUE NOT NULL`. I'm not sure what the combination of that `SERIAL` + `REFERENCES user(id)` will do (this is my first experience working with postgres)... i believe `SERIAL` attempts to auto-increment the values, which would we would not want here.

Reply:
Otherwise, this looks good!

Reply:
You're right - switched it to INTEGER to avoid any side effects